src/doc/rustc/src/platform-support/win7-windows-msvc.md

```diff
@@ -1,8 +1,8 @@
-# *-win7-windows-msvc
+# \*-win7-windows-msvc
 
 **Tier: 3**
```

Comment:
Suggestion (drive-by): could you also just add

- `x86_64-win7-windows-msvc`, and
- `i686-win7-windows-msvc`

for search purposes too?

Reply:
Done.